src/Raven.Studio/wwwroot/Content/css/pages/indexes.less

```diff
@@ -85,6 +88,22 @@
             font-size: 14px;
         }
 
+        .index-type-icon {
+            text-align: center;
+            width: 21px;
+            margin-right: @grid-gutter-width/4;
+            align-self: flex-start;
+            
+            .tooltip {
```

Comment:
don;t we have set this up globally? Do we need to override this only here?

Reply:
1. I think we should override here 
    I see other places (such as _dashboard.less, metrics.less, index-performance.less_ ) that have 
   other overriding values

2. Overriding is needed - as can be seen in the images

3. Should we **unify** all over studio (i.e. tooltip-narrow -vs- tooltip-wide) ?

![image](https://user-images.githubusercontent.com/17376974/71189433-824b6880-228b-11ea-87ab-18193f491002.png)
![image](https://user-images.githubusercontent.com/17376974/71189456-88d9e000-228b-11ea-9aa1-ec9c952a43cf.png)